README.md

```diff
@@ -1,6 +1,14 @@
 # lambda-sc-cost-meter
 Lambda to report Service Catalog costs to the AWS Marketplace
 
+## Description
+
+This app will look for resources in an AWS account that contain tags indicating that
```

Comment:
"will look for **EC2 instances** in an AWS account" (unless API is changed to get all resources)

Reply:
the app now gets cost of all resources, not just EC2.